geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreTest.scala

```diff
@@ -72,6 +74,21 @@ class AccumuloDataStoreTest extends Specification with TestWithMultipleSfts {
       ds.getSchema(defaultSft.getTypeName) mustEqual defaultSft
     }
 
+    "create a schema with keywords" in {
+
+      val sftWithKeywords: SimpleFeatureType = createNewSchema("name:String", dtgField = None)
+      val keywords: java.util.Set[String] = Set("keywordA", "keywordB", "keywordC")
+      sftWithKeywords.getUserData.put("geomesa.keywords", keywords.mkString(keywordsDelimitter))// Put keywords in userData
+
+      val spec = SimpleFeatureTypes.encodeType(sftWithKeywords, true)
+      val newType = SimpleFeatureTypes.createType("keywordsTest", spec)
+      ds.createSchema(newType)
+
+      val fs = ds.getFeatureSource(newType.getTypeName)
+
+      (fs.getInfo.getKeywords containsAll keywords) mustEqual true
```

Comment:
[minor] this would work better as `fs.getInfo.getKeywords must containAllOf(keywords)` (syntax may be slightly off).
The advantage is that if that fails, you will get a more descriptive error message (e.g. list(foo, bar) contains foo and bar but does not contain baz), vs as currently written you would just get a 'value is false' failure.